MAiD - Case Management App/force-app/main/default/objects/S1_HC_2023__c/fields/IND_Q05A__c.field-meta.xml

```diff
@@ -1,9 +1,9 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <CustomField xmlns="http://soap.sforce.com/2006/04/metadata">
     <fullName>IND_Q05A__c</fullName>
+    <deprecated>false</deprecated>
     <externalId>false</externalId>
-    <formula>IF( ISBLANK(Form_1634__r.Case__r.If_yes_Indigenious_heritage__c) , 0, IF( INCLUDES(Form_1634__r.Case__r.If_yes_Indigenious_heritage__c, &apos;First Nations&apos;) , 1, 2))</formula>
-    <formulaTreatBlanksAs>BlankAsZero</formulaTreatBlanksAs>
+    <formula>IF( ISBLANK(Form_1634__r.Case__r.If_yes_Indigenious_heritage__c) , 0, IF( INCLUDES(Form_1634__r.Case__r.If_yes_Indigenious_heritage__c, &quot;First Nations&quot;) , 1, 2))</formula>
```

Comment:
"If, yes Indigenous heritage" contains "First Nations" then cell is "1" OR, if No, then cell is "2" OR, if blank, cell is "0"

Actual Function:
- Where If_yes_Indigenious_heritage__c is blank it returns 0
- If_yes_Indigenious_heritage__c containing "First Nations" returns 1
- Default response when not blank and not "First Nations" returns 2

My reading of this is as follows:
- Blank is not First Nations (0)
- Unspecified First Nations where just First Nations is entered is (1)
- Any other specific First Nations entry returns (2)

Function is performing as expected. @RanadheerRG